### PR TITLE
Removed open parenthesis

### PR DESCRIPTION
**Pull request by hakito**
# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/framework/serialization/how-to-serialize-an-object-as-a-soap-encoded-xml-stream.md [hunk 1]
```diff
@@ -65,14 +65,14 @@ manager: "erikre"
   
 ```vb  
 ' Serializes a class named Group as a SOAP message.  
-Dim myTypeMapping As XmlTypeMapping = (New SoapReflectionImporter(). _  
+Dim myTypeMapping As XmlTypeMapping = New SoapReflectionImporter(). _  
 ImportTypeMapping(GetType(Group))
```
Comment (svick) [inline docs/framework/serialization/how-to-serialize-an-object-as-a-soap-encoded-xml-stream.md]: I think this line should be indented, to make it clearer that it's not a separate statement. Do you want to add that change to this PR?

Alternatively, you could just put the statement on a single line, like the C# version does.

**Comment by svick** [comment]
The partial examples on lines 42 and 48 also have this issue. I think those should be fixed too.

**Review by guardrex — APPROVED** [review]
The indentation isn't explicitly stated everywhere but implicitly shown in various examples ... for example, https://docs.microsoft.com/dotnet/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code

Also, we can go with implicit line continuation here: https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/statements#implicit-line-continuation

... but this was done as a patch, so @svick let's go ahead and accept the change and then put the indentation/continuation character change update on a new PR.

Thanks @hakito for fixing these! :rocket:

**Comment by svick** [comment]
> but this was done as a patch

You can add a commit to a patch by editing the file in the patch branch using the GitHub UI. Though doing that is fairly unintuitive, so I'm fine if the policy is "additional improvements to patch PRs are generally done in a separate PR".

**Comment by GuardRex** [comment]
> additional improvements to patch PRs are generally done in a separate PR

Correct ... this was recently explained to me after I asked someone to make an additional minor change that wasn't required to accept a community PR. The general rule is that if a patch from a community member is good to go ahead and :shipit: ... then if we need a small touch-up for some new little thingy to just shoot in a follow-up PR. I opened it on #2475.

**Pull request merged**